src/utils/formik-helpers/project-form-helpers.ts

```diff
@@ -26,8 +26,51 @@ export const getProjectTabErrors = (errors: FormikErrors<CristinProject>, touche
       touched
     ),
     [ProjectTabs.Details]: getErrorMessages<CristinProject>(Object.values(ProjectDetailsFieldNames), errors, touched),
-    [ProjectTabs.Contributors]: getErrorMessages<CristinProject>(Object.values(ProjectContributorsFieldNames), errors),
+    [ProjectTabs.Contributors]: getErrorMessages<CristinProject>(
+      Object.values(ProjectContributorsFieldNames),
+      errors,
+      touched
+    ),
   };
 
   return tabErrors;
 };
+
+const mandatoryFieldsTab0 = {
+  title: true,
+  startDate: true,
+  endDate: true,
+};
+
+const mandatoryFieldsTab1 = {
+  coordinatingInstitution: { id: true },
+};
+
+const mandatoryFieldsTab2 = {
+  contributors: true,
+};
+
+const mandatoryFieldsTab3 = {};
+
+export const getTouchedFields = (tabNumber: number) => {
```

Comment:
burde denne begrense input til å matche gyldig type?
```suggestion
export const getTouchedFields = (tabNumber: ProjectTabs) => {
```

Reply:
Ser nå at det ikke vil stemme siden denne går ett tall over antall tabs vi har. Vet ikke om det er by-design ikke, så det kan du ev. ta en gløtt på. Rent intuitivt virker det rart at man her tar inn `tabNumber` som ikke eksisterer i wizarden, men om det er en bevisst ting er det greit!